homeassistant/components/evohome.py

```diff
@@ -0,0 +1,144 @@
+"""Support for Honeywell evohome (EMEA/EU-based systems only).
+
+Support for a temperature control system (TCS, controller) with 0+ heating
+zones (e.g. TRVs, relays) and, optionally, a DHW controller.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.evohome/
```

Comment:
URL is wrong.

Reply:
What should it be? `https://www.home-assistant.io/components/climate.evohome/`?

Reply:
The web page doesn't exist yet - it is a PR - home-assistant.io/6252

Reply:
It should be `https://home-assistant.io/components/evohome/`. It's ok to add it here before the PR is merged.

Reply:
Just to be clear, there should be two pages, one for the component and one for the platform.

Reply:
Ah!

Reply:
OK, done.